{"uuid": "c85e1c38-9063-4f12-ac16-6ff51540122e", "children": ["c110a25b-be49-45e7-a1be-6958c68e8214"], "befores": [{"name": "_Class__pytest_setup_class", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: cannot find Chrome binary\n\n", "trace": "  File \"D:\\python\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 957, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 783, in call_fixture_func\n    res = next(it)\n  File \"D:\\python\\lib\\site-packages\\_pytest\\python.py\", line 731, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"D:\\python\\lib\\site-packages\\_pytest\\python.py\", line 669, in _call_with_optional_argument\n    func(arg)\n  File \"D:\\Jenkins\\workspace\\AotoTest_JOB3\\test_case\\test_0003_add_project.py\", line 20, in setup_class\n    self.driver =  RedmineLogin().log_redmine()\n  File \"D:\\Jenkins\\workspace\\AotoTest_JOB3\\common\\redmine_login.py\", line 19, in log_redmine\n    driver = webdriver.Chrome()\n  File \"D:\\python\\lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 81, in __init__\n    desired_capabilities=desired_capabilities)\n  File \"D:\\python\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 157, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"D:\\python\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 252, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"D:\\python\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"D:\\python\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1585277992264, "stop": 1585277994831}], "start": 1585277992264, "stop": 1585277994871}